ANMN/burst_averaged_product/generate_ctd_nc_file_gatt

```diff
@@ -0,0 +1,4 @@
+[global_attributes]
+abstract = SAIMOS NRS mooring location. This mooring collects times series data of water currents (x,y,z), CTD (fluorescence, PAR, DO & turbidity). SARDI maintains the SAIMOS mooring network. Data from bursts have been cleaned and averaged to create data products. This file is one such product.
```

Comment:
abstract and keywords should not be hard-coded. You can just append the relevant info (e.g. last two sentences in abstract) to whatever's in the input file.